### PR TITLE
Database/week1

### PR DESCRIPTION
**Pull request by hamedveloper**
I tried to remove JS3/W3 in this PR however I could not!!!

### CODE REVIEW

File: databases/week1/database.sql [hunk 1]
```diff
@@ -0,0 +1,70 @@
+--1.Find out how many tasks are in the task table
+
+SELECT COUNT(*)
+FROM HYF.task;
```
Comment (khjcph) [inline databases/week1/database.sql]: curious why you use `HYF.task` here and not just `task` like in the other exercises?

File: databases/week1/database.sql [hunk 2]
```diff
@@ -0,0 +1,70 @@
+--1.Find out how many tasks are in the task table
+
+SELECT COUNT(*)
```
Comment (khjcph) [inline databases/week1/database.sql]: Maybe use an alias? `SELECT COUNT(*) AS tasks_count` for example.

File: databases/week1/database.sql [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+FROM HYF.task;
+
+--.Find out how many tasks in the task table do not have a valid due date
+
+SELECT COUNT(*)
```
Comment (khjcph) [inline databases/week1/database.sql]: Maybe an alias here too?

File: databases/week1/database.sql [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+FROM task
+WHERE due_date IS NULL;
+
+--3.Find all the tasks that are marked as done
+
+SELECT *
+FROM task t
+    INNER JOIN status s
+    ON t.status_id=s.id
+WHERE s.name="Done";
```
Comment (khjcph) [inline databases/week1/database.sql:18]: I think this is fine, but I am curios why you need an alias here? Why not:

    SELECT *
    FROM task
      INNER JOIN status
      ON task.status_id = status.id
    WHERE status.name = "Done";

File: databases/week1/database.sql [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+--4.Find all the tasks that are not marked as done
+
+SELECT *
+FROM task t
+    INNER JOIN status s
+    ON t.status_id=s.id
+WHERE s.name<>"Done";
```
Comment (khjcph) [inline databases/week1/database.sql:26]: Same as above; do you need to alias here?

**Comment by khjcph** [comment]
Great work! I left a few comments for you to look at. Some general feedback

- Try to think about readability when aliasing. This is a small thing but
    
        SELECT COUNT(status.name) AS count, status.name AS status
        FROM task
          INNER JOIN status
          ON task.status_id = status.id
        GROUP BY status.name
        ORDER BY count DESC;
    is a bit easier to read than

        SELECT COUNT(s.name) AS count, s.name AS status
        FROM task t
          INNER JOIN status s
          ON t.status_id = s.id
       GROUP BY s.name
       ORDER BY count DESC;
    this is just a personal preference though.

- I think you committed the javascript files here by mistake?
- When creating commits, try to give them some meaningful names. They are all just called `first commit` :). But the exercise is not about GIT, so np. 

But again, good job! 👍

**Comment by hamedveloper** [comment]
Thank you for your comments. I wanted just to make it simple.

**Pull request merged**